meta-resin-common/recipes-support/resin-persistent-logs/resin-persistent-logs/resin-persistent-logs

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ "$PERSISTENT_LOGGING" == "true" ] && [ ! -d /var/log/journal ]; then
+    systemctl start var-log-journal.mount
+    journalctl --flush
+    echo "resin-persistent-logs: Persistent logging activated."
+elif [ "$PERSISTENT_LOGGING" == "true" ] && [ -d /var/log/journal ]; then
+    echo "resin-persistent-logs: Persistent logging already activated."
+    journalclt --flush
+elif [ "$PERSISTENT_LOGGING" == "false" ] && [ -d /var/log/journal ]; then
+    echo "resin-persistent-logs: Persistent logging was deactivated but system reboot is needed."
```

Comment:
So what happens in this case? Logs are still written to disk until system reboot?